client_cli/README.md

```diff
@@ -43,71 +49,52 @@ Full description and list of commands detailed in `./iroha_client_cli --help`.
 
 ### Create new Domain
 
-Let's start with domain creation. We need to provide `register` command first, 
-following by entity type (domain in our case) and list of required parameters.
-For domain entity we only need `id` parameter as a string.
+Let's start with domain creation. To create a domain, you need to specify the entity type first (`domain` in our case) and then the command (`register`) with a list of required parameters.
+For domain entity you only need to provide the `id` parameter as a string.
 
 ```bash
 ./iroha_client_cli domain register --id="Soramitsu"
 ```
 
+Now you have a domain without any accounts.
+
 ### Create new Account
 
-Right now we have the only domain without any accounts, let's fix it.
-Like in the previous example, we need to define account name, it is done using `id` flag.
-We also give a `key` argument with account's public key as a double-quoted
-string value.
+Let's create a new account. Like in the previous example, specify the entity type (`account`) and the command (`register`). Then define the account name with the `id` flag.
+You also need to provide the `key` argument with the account's public key as a double-quoted
+string.
 
 ```bash
 ./iroha_client_cli account register --id="White Rabbit@Soramitsu" --key=""
 ```
 
 ### Mint Asset to Account
 
-Okay, it's time to give something to our Account. We will add some Assets quantity to it.
-This time we need to register an Asset Definition first and then add some Assets to the account.
-As you can see, we use new command `asset` and it's subcommands `register` and `mint`. 
-Every asset has its own value type, here we define domain as quantity (integer/number).
+It's time to give something to the Account you created. Let's add some Assets quantity to it.
+
+This time, you need to register an Asset Definition first and then add some Assets to the account. To do this, specify the `asset` entity and then use the `register` and `mint` commands respectively. Every asset has its own value type. In this example, it is defined as quantity (integer/number).
 
 ```bash
 ./iroha_client_cli asset register --id="XOR#Soramitsu" --value-type=Quantity
 ./iroha_client_cli asset mint --account="White Rabbit@Soramitsu" --asset="XOR#Soramitsu" --quantity=1010 
 ```
 
+You created `XOR#Soramitsu` asset of type `Quantity` and then gave `1010` units of this asset to the `White Rabbit@Soramitsu` account.
+
 ### Query Account Assets Quantity
 
 Because distributed systems heavily relay on the concept of eventual consistency and Iroha works in Consensus between peers, your requests may or may not be processed
```

Comment:
```suggestion
Because distributed systems heavily rely on the concept of _eventual_ consistency and Iroha works by awaiting consensus between peers, your request is not guaranteed to be processed (or be accepted) even if it is correctly formed. 
```